### PR TITLE
Added extension method to handle the HTTP_PLATFORM_PORT

### PR DESCRIPTION
**Pull request by davidfowl**
/cc @JunTaoLuo @Tratcher

### CODE REVIEW

File: src/Microsoft.AspNet.IISPlatformHandler/IISWebApplicationBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class IISWebApplicationBuilderExtensions
+    {
+        public static IWebApplicationBuilder UseIISPlatformHandler(this IWebApplicationBuilder webApplicationBuilder)
```
Comment (Tratcher) [inline src/Microsoft.AspNet.IISPlatformHandler/IISWebApplicationBuilderExtensions.cs]: IWebApplicationBuilder.UseIISPlatformHandler needs to be searchably distinct from IApplicationBuilder.UseIISPlatformHandler. UseIISPlatformHandlerUrl?

**Comment by Tratcher** [comment]
I was also testing this code path yesterday and found that it fails unless we first remove the same logic from Hosting. Otherwise the address is double registered and kestrel crashes.

**Comment by Tratcher** [comment]
https://github.com/aspnet/IISIntegration/issues/52

**Comment by davidfowl** [comment]
Closing

**Comment by davidfowl** [comment]
:shipit:

**Pull request closed without merging**